firmware/integration/rusefi_config.txt

```diff
@@ -744,15 +744,20 @@ custom maf_sensor_type_e 4 bits, S32, @OFFSET@, [0:7], @@maf_sensor_type_e_enum@
 
     output_pin_e boostControlPin;
 	pin_output_mode_e boostControlPinMode;
+
```

Comment:
not useful changes in rusefi_config.txt so please revert this file - this would revert it from PR

Reply:
I only edited something to resolve conflict.. How do I revert it?

Have tried to delete it as well..